Makefile

```diff
@@ -23,8 +23,14 @@ else
 endif
 
 REPOPATH ?= k8s.io/minikube
+K8S_VENDOR_PATH := $(REPOPATH)/vendor/k8s.io/kubernetes
 export GO15VENDOREXPERIMENT=1
 
+# Update this to match the vendored version of k8s in godeps.json
+K8S_VERSION := v1.3.0-alpha.3-838+ba170aa191f8c7
+# Set the version information in kubernetes.
+LD_FLAGS = "-s -w -X $(K8S_VENDOR_PATH)/pkg/version.gitVersion=$(K8S_VERSION) -X $(K8S_VENDOR_PATH)/pkg/version.gitCommit=$(shell git rev-parse HEAD)"
```

Comment:
> version.gitCommit=$(shell git rev-parse HEAD)

HEAD of localkube isn't the same as gitCommit for Kubernetes.
I think we should query this dynamically from `Godeps.json`
I also think there's more `-X` parameters we should set